[BPO-30924] [BPO-30925] doc-files separate line and additional files patches

warthog9 opened the pull request:
doc-files Breakup should break files up into multiple lines, as per RPM expectation

when using %doc section of the RPM spec, it expects multiple files
to be listed each on their own line.  Currently what gets passed in
is:

%doc file1 file2 file3

which throws:
	error: More than one file on a line: file1
	error: More than one file on a line: file2
	error: More than one file on a line: file3

This breaks the files up into seperate lines:

%doc
file1
file2
file3

which works as expected with multiple files

Signed-off-by: John 'Warthog9' Hawley <jhawley@vmware.com>
Signed-off-by: John 'Warthog9' Hawley <warthog9@eaglescrag.net>

warthog9 committed:
doc-files should break files up into multiple lines, as per RPM expectation

when using %doc section of the RPM spec, it expects multiple files
to be listed each on their own line.  Currently what gets passed in
is:

%doc file1 file2 file3

which throws:
	error: More than one file on a line: file1
	error: More than one file on a line: file2
	error: More than one file on a line: file3

This breaks the files up into seperate lines:

%doc
file1
file2
file3

which works as expected with multiple files

Signed-off-by: John 'Warthog9' Hawley <jhawley@vmware.com>
Signed-off-by: John 'Warthog9' Hawley <warthog9@eaglescrag.net>

the-knights-who-say-ni commented:
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA. This is necessary for legal reasons before we can look at your contribution. Please follow these steps to help rectify the issue:

1. If you don't have an account on b.p.o, please [create one](http://bugs.python.org/user?@template=register)
2. Make sure your GitHub username is listed in ["Your Details" at b.p.o](https://cloud.githubusercontent.com/assets/2680980/23276970/d14a380c-f9d1-11e6-883d-e13b6b211239.png)
3. If you have not already done so, please sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/). The "bugs.python.org username " requested by the form is the "Login name" field under "Your Details".
4. If you just signed the CLA, please **wait at least one US business day** and then check "Your Details" on [bugs.python.org](https://bugs.python.org/) to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)
5. Reply here saying you have completed the above steps


Thanks again to your contribution and we look forward to looking at it!

warthog9 committed:
Duplicate doc-files functionality, to inject files into bdist_rpm

This duplicates the existing functionality from doc-files, however
it includes the files in the %files section as opposed to the %docs
section of the RPM

Signed-off-by: John 'Warthog9' Hawley <jhawley@vmware.com>
Signed-off-by: John 'Warthog9' Hawley <warthog9@eaglescrag.net>

kushaldas commented:
Few things are missing: 

- A bug in bugs.python.org
- Update the title of the PR based on that
- A test case for the same
- Docs update to reflect this change
